src/common/Sample.cpp

```diff
@@ -46,12 +46,16 @@ bool Sample::load(const char* file_name)
       *comma = 0;
    }
 
+#if WINDOWS
    if (!stricmp(extension, "wav"))
+#else
+   if (!strcasecmp(extension, "wav"))
```

Comment:
stupid question: is there a standard for this function?

Reply:
There isn't! Annoying right? If you squint you can convince yourself that c strings are memory blocks so ideas like 'case' are encoding specific, then get into wchar and encodings (and maybe even ebcdic!). So everyone needs to define one as the other or vice versa somewhere (which @whydoubt noted over on #173).

I'm not quite sure where that re-definition has gone in this PR, either. @jsakkine ?

Reply:
And as I noticed over on the issue I read this diffs backwards. We should not add a platform specific ifdef every time we do a stricmp. I added 3 more to squash a bug yesterday for instance 

So if we are taking out of the premake it’s gotta go in a header somewhere and then we can just code to stricmp everywhere. I don’t think we should merge this change